metals/src/main/scala/scala/meta/internal/parsing/Trees.scala

```diff
@@ -102,9 +100,14 @@ final class Trees(
             )
           case Parsed.Success(tree) =>
             trees(path) = tree
-            List()
+            List.empty
+          case other =>
+            scribe.error(
```

Comment:
I don;t think we need to log it since it's not possible currently to get any other result.